types/test.tsx

```diff
@@ -115,13 +115,32 @@ export function wrappedRenderB(
   ui: React.ReactElement,
   options?: pure.RenderOptions,
 ) {
-  const Wrapper: React.FunctionComponent = ({children}) => {
+  const Wrapper: React.FunctionComponent<{children?: React.ReactNode}> = ({
```

Comment:
While `FunctionComponent` adds implicit children, its `propTypes` do not so we didn't catch that issue. Here we make sure all statics assume the same props. Also makes sure this doesn't break once we get rid of implicit children in `React.FunctionComponent`.